### PR TITLE
Fgw doc improvements

### PR DESCRIPTION
**Pull request by JonathanLink**

### CODE REVIEW

File: docs/config.json [hunk 1]
```diff
@@ -1,17 +0,0 @@
-{
-  "title": "Crafty",
-  "tagline": "The task runner that binds them all",
-  "author": "Swissquote Bank SA",
-  "html": {
-    "theme": "swissquote",
-    "repo": "swissquote/crafty",
-    "google_analytics": "UA-113355094-2",
```
Comment (onigoetz) [inline docs/config.json:8]: not sure why you deleted the whole file, I think just removing this line was enough

**Review by onigoetz — CHANGES_REQUESTED** [review]
mostly ok, just one question on the config.json change

File: docs/config.json (hunk 1 above)
Comment (JonathanLink) [reply docs/config.json:8]: done

**Review by onigoetz — APPROVED** [review]
LGTM

**Pull request merged**